test/e2e/case20_informonly_test.go

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) 2023 Red Hat, Inc.
+// Copyright Contributors to the Open Cluster Management project
+
+package e2e
+
+import (
+	"errors"
+	"os/exec"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"open-cluster-management.io/config-policy-controller/test/utils"
+)
+
+const (
+	case20PolicyName              string = "case20-policy-informonly"
+	case20PolicyYaml              string = "../resources/case20_policy_informonly/case20-parent-policy.yaml"
+	case20ConfigPlcName           string = "create-configmap"
+	case20PolicyNoRemediationName string = "case20-policy-informonly-no-remediationaction"
+	case20PolicyNoRemediationYaml string = "../resources/case20_policy_informonly/" +
+		"case20-parent-policy-noremediation.yaml"
+)
+
+func checkInformAction() {
+	By("Checking template policy remediationAction")
+	Eventually(func() interface{} {
+		plc := utils.GetWithTimeout(clientManagedDynamic, gvrConfigurationPolicy,
+			case20ConfigPlcName, clusterNamespace, true, defaultTimeoutSeconds)
+
+		return plc.Object["spec"].(map[string]interface{})["remediationAction"]
+	}, defaultTimeoutSeconds, 1).Should(Equal("inform"))
+}
+
+var _ = Describe("Test 'InformOnly' ConfigurationPolicies", Ordered, func() {
```

Comment:
The `BeforeAlls` and `AfterAlls` being defined in different places make this tricky to understand.

`BeforeAll` is for setup that multiple tests will use. In this case, I don't think it's a good fit for creating the parent policy, since each test uses a different parent policy. I think the creation of that policy could just be inside the `It` for the test.

`AfterAll` is for cleanup that can happen after all the tests here run. It is good for cleaning up the policies, because it is guaranteed to run, even if a test fails. But in this case I think a better `AfterAll` would be to clean up both policies and templates used by these tests... Just be sure that if a policy or template isn't present, that doesn't cause an error.

I think if you try that, the tests could look more like:
- `AfterAll`: cleanup everything used in this test file
- `It`: 
    - create a parent policy with remediationAction=enforce in the template
    - check the template has remediationAction=inform
- `It`:
    - create a parent policy with the remediationAction unset in the template
    - check the template has remediationAction=inform